### PR TITLE
[MIG] mis_builder_demo: migration to 17.0

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: mis_builder_budget/models/mis_budget_by_account_item.py [hunk 1]
```diff
@@ -5,7 +5,11 @@
 
 
 class MisBudgetByAccountItem(models.Model):
-    _inherit = ["mis.budget.item.abstract", "prorata.read_group.mixin"]
+    _inherit = [
+        "mis.budget.item.abstract",
+        "prorata.read_group.mixin",
+        "analytic.mixin",
```
Comment (sbidoul) [inline mis_builder_budget/models/mis_budget_by_account_item.py]: Adding the analytic mixin goes beyond the scope of a migration. Can we do this in another PR?

Comment (dreispt) [reply mis_builder_budget/models/mis_budget_by_account_item.py]: Sure: https://github.com/OCA/mis-builder/pull/634

**Review by sbidoul — APPROVED** [review]
